docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -218,6 +218,10 @@ A sample HTTPRoute status is shown below:
         sectionName: http
   ```
 
+#### Services
+
+Starting from AKO 1.12.1 we will support Services with multiple ports for Gateways too.
```

Comment:
Can we create `ChangeLog` file so that it will be easier to track features added/changed/removed/known issue as per Gateway release?

Reply:
Can we convert all sentence having `we will ` or `our implementation` to `will be` or `AKO Gateway will` ?